src/spikeinterface/extractors/cellexplorersortingextractor.py

```diff
@@ -118,7 +118,8 @@ def __init__(
         spike_times = spikes_data["times"]
 
         # CellExplorer reports spike times in units seconds; SpikeExtractors uses time units of sampling frames
-        unit_ids = unit_ids[:].tolist()
+        unit_ids = unit_ids[:].astype(int).tolist()
+        unit_ids = [str(unit_id) for unit_id in unit_ids]
```

Comment:
Why are we going to str in this extractor ?
Why do we need int and then str and why not directly str ?

Reply:
right, we only need the second line

Reply:
done